README.md

```diff
@@ -14,39 +14,48 @@ more effort to download and install so we supply a mock compiler service for cas
 the UI application itself. The easiest way to spin up all of this
 stuff with correct wiring is to use the provided docker-compose
 configuration.
 
-The default docker-compose configuration requires the fully-functional Compiler
-service converter image. To bypass installation of this compiler image use the mock compiler
-configuration option.
-
-How to chose the option that's right for you: If you are not concerned with TeX
-compilation, or you have limited disk space, you will want the mock compiler option.
-If you are concerned with evaluating TeX compilation output and/or the submission
-'process' step under normal operating conditions you will want to install the
-actual Compiler service's converter image.
-
-## Setup required for both mock and full compilation options - AWS settings
+```bash
+mkdir /tmp/foo          # Mock compiler service will use this.
+export DIND_SOURCE_ROOT=/tmp/foo
+export AWS_ACCESS_KEY_ID=fookey  AWS_SECRET_ACCESS_KEY=foosecretkey
+cd /path/to/arxiv-submission-ui
+docker-compose -f docker-compose-mock-compiler.yml pull     # Pulls in images that you might not have already.
+docker-compose -f docker-compose-mock-compiler.yml build    # Builds the submission UI
+docker-compose -f docker-compose-mock-compiler.yml up
+```
 
-One of the back-end services we run is 'localstack' which is a fully functional
-local AWS stack. While this local AWS stack implementation doesn't actually
-check credentials you must have the environment variables set for the service
-to work properly.
+After starting the "up" command, scroll down past about 400 lines of log messages.
+A bootstrap process will generate a bunch of users who are authorized to submit things.
+Cut and paste one of those JWTs. It will look something like this:
+```
+arxiv-submission-ui     | 1 picoline2058@gmail.com _JWT_STARTS_HERE_AND_GOES_ON_FOR_SEVERAL_LINES_
+```
+You will need to pass the JWT in the ``Authorization`` header.
+This is a valid session so you can use the submission UI.
+For Chrome, try the [Requestly](https://www.requestly.in/) plugin, where you:
+- Create a rule to modify headers
+- Add a request header of "Authorization", with a value as the JWT you found in the logs.
+- Add a value for "request url contains" as localhost:8000.
 
-You may set these to your real AWS credentials or simply set them to arbitrary
-dummy values.
+You should be able to access the UI at http://localhost:8000. If you don't
+get a response right away, the UI is probably still waiting for something
+to come up. If you are redirected to the production ArXiv.org login page,
+then your browser is not sending a valid JWT to localhost.
 
+To stop the service, and remove the docker compose containers, you can Ctrl-C or "stop", and then "rm":
 ```bash
-export AWS_ACCESS_KEY_ID=fookey
-export AWS_SECRET_ACCESS_KEY=foosecretkey
+docker-compose -f docker-compose-mock-compiler.yml stop
+docker-compose -f docker-compose-mock-compiler.yml rm -v
```

Comment:
I also run 'docker system prune' after 'rm -v' to remove the stopped containers. Might be worth adding here.

Reply:
Good idea, adding that now.